src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -300,9 +300,12 @@ object DockerPlugin extends AutoPlugin {
               "--push"
             ) ++ dockerBuildOptions.value :+ "."
           else dockerExecCommand.value
-        alias.foreach { aliasValue =>
-          publishDocker(context, execCommand, aliasValue.toString, log, multiplatform)
-        }
+        // For multiplatform builds, the alias are part of the `dockerBuildOptions` already.
+        if (multiplatform) publishDocker(context, execCommand, dockerAlias.value.tag.getOrElse(""), log, multiplatform)
```

Comment:
That empty string is weird, but the `publishDocker` method does not use it for multiplatform - so this works :grin: